unreal_modloader/src/mod_config.rs

```diff
@@ -15,6 +15,8 @@ struct ModConfig {
     selected_game_platform: Option<String>,
     refuse_mismatched_connections: bool,
     current: ModsConfigData,
+    #[serde(default)]
+    trusted_mods: Vec<Vec<u8>>,
```

Comment:
store as hex?

Reply:
it is hex, there is just no need for converting back and forth as noone should edit this field

Reply:
I just feel like a json file with [0, 1, 2 ...] as a hash would be weird.

Reply:
fair point